Creative Bakery

lisabogdantseva opened the pull request:
[DEMO LINK](https://lisabogdantseva.github.io/layout_creativeBakery/)

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,363 @@
+        </div>
+      </div>
+      <section class="products" id="products">
+        <h2 class="
+          section-title
+          section-title--align--left
+          products__title">What we bake</h2>
```
```suggestion
        <h2
          class="
            section-title
            section-title--align--left
            products__title"
        >
          What we bake
        </h2>
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,363 @@
+      <div class="container">
+        <div class="main__content">
+          <section class="about-us" id="about-us">
+            <h2 class="
+            section-title
+            section-title--align--left
+            about-us__title">Few words About us</h2>
```
The same as above

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,363 @@
+          </div>
+          <section class="contact-us" id="contact-us">
+            <div class="footer__contact-wrapper">
+              <h2 class="
+              section-title
+              section-title--align--left
+              section-title--contact
```
The same

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,363 @@
+                    Call us
+                  </li>
+                  <li class="nav__item">
+                    <a class="nav__link" href="tel:(629) 555-0129 ">(629) 555-0129</a>
```
Need to fix format phone everywhere only in `href`
```suggestion
                    <a class="nav__link" href="tel:6295550129">(629) 555-0129</a>
```

volodymyr-soltys97 commented on src/styles/blocks/about-us.scss:
```diff
@@ -0,0 +1,90 @@
+      width: 718px;
+      top: 0%;
+      left: 26%;
+    }
+
+  }
```
```suggestion
    }
  }
```

volodymyr-soltys97 commented on src/styles/blocks/footer.scss:
```diff
@@ -0,0 +1,117 @@
+
+  &__creadentials {
+    font-size: 16px;
+    font-style: normal;
```
It's default value, you can just remove it
```suggestion
```

volodymyr-soltys97 commented on src/styles/blocks/hero.scss:
```diff
@@ -0,0 +1,182 @@
+  }
+
+  &__btn {
+    box-sizing: border-box;//?
```
Move it to * selector
```suggestion
```

volodymyr-soltys97 commented on src/styles/blocks/hero.scss:
```diff
@@ -0,0 +1,182 @@
+  }
+
+  &__btn:hover .icon--cookie--white,
+  &__btn:focus .icon--cookie--white {
+
+    display: block;
+
+  }
```
```suggestion
  &__btn:focus .icon--cookie--white {
    display: block;
  }
```

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your code

lisabogdantseva commented:
Thank you for the comments!

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,371 @@
+          </section>
+        </div>
+      </div>
+    </main>
+    <footer class="footer" id="contacts">
```
```suggestion
    </main>
    
    <footer class="footer" id="contacts">
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,371 @@
+                  </li>
+                </ul>
+              </div>
+
+
```
There should only be one blank line between blocks for better readability, check it everywhere
```suggestion

```

volodymyr-soltys97 commented on src/styles/blocks/menu.scss:
```diff
@@ -0,0 +1,29 @@
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 24px;
+
+  }
```
```suggestion
    gap: 24px;
  }
```

volodymyr-soltys97 commented on src/styles/blocks/nav.scss:
```diff
@@ -0,0 +1,32 @@
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      gap: 4px;
+
+    }
+  }
+
+}
```
```suggestion
      gap: 4px;
    }
  }
}
```

volodymyr-soltys97 commented on src/styles/blocks/products.scss:
```diff
@@ -0,0 +1,141 @@
+
+  @include onDesktopSm {
+    margin-bottom: 178px;
+  }
+  &__title {
+    margin-bottom: 24px;
+
+  }
```
```suggestion
  }

  &__title {
    margin-bottom: 24px;
  }
```

volodymyr-soltys97 commented on src/styles/blocks/products.scss:
```diff
@@ -0,0 +1,141 @@
+
+    @include onTablet {
+      width: 100%;
+    }
+
+  }
```
```suggestion
    }
  }
```

lisabogdantseva commented:
Thank you for the advice! I've changed everything, but I hope I didn't overdo in html (I added blank lines between some siblings of the block as well, as was in the checklist - https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

volodymyr-soltys97 reviewed (approved):
Well done 🔥
